pkg/virt-handler/vm.go

```diff
@@ -2010,7 +2010,14 @@ func (d *VirtualMachineController) heartBeat(interval time.Duration, stopCh chan
 				log.DefaultLogger().Reason(err).Errorf("Can't determine date")
 				return
 			}
-			data := []byte(fmt.Sprintf(`{"metadata": { "labels": {"%s": "true"}, "annotations": {"%s": %s}}}`, v1.NodeSchedulable, v1.VirtHandlerHeartbeat, string(now)))
+
+			kubevirtSchedulable := "true"
```

Comment:
I see this as being "stringly typed". Although I don't have a real objection against this I find this looking odd.

Reply:
It is a label. Labels are of type `map[string]string`.